deploy/ecs/gprofiler_task_definition.json

```diff
@@ -0,0 +1,66 @@
+{
+    "requiresCompatibilities": [
+        "EC2"
+    ],
+    "containerDefinitions": [
+        {
+            "name": "granulate-gprofiler",
+            "image": "index.docker.io/granulate/gprofiler:latest",
+            "memory": "800",
+            "memoryReservation": "256",
+            "cpu": "500",
+            "essential": true,
```

Comment:
What's this

Reply:
If the gProfiler container fails, then the deployment itself would count as a failure. Since this is the only container in the task def, it makes sense

Reply:
k